enterprise/backend/src/metabase_enterprise/workspaces/api.clj

```diff
@@ -14,16 +13,28 @@
 
 ;;; schemas
 
-(mr/def ::stuff
-  :keyword
-  #_[:string {:min 1 :max 254}])
+(def ^:private type->t2-model {:transform :model/Transform})
+
+;; The key we use to group entities of the same type in requests and responses.
+(def ^:private type->grouping {:transform :transforms})
+
+(mr/def ::entity-type [:enum :transform])
+
+(mr/def ::entity-grouping [:enum :transforms])
+
+;; Entities that live within the Workspace
+(mr/def ::downstream-entity
+  [:map
+   [:id ms/PositiveInt]
+   [:type ::entity-type]
+   [:name :string]])
 
 (def CreateWorkspace
   "Schema for creating a new workspace"
   [:map
    [:name [:string {:min 1}]]
-   [:database_id :int]
-   [:stuffs [:map-of ::stuff [:sequential ms/PositiveInt]]]])
```

Comment:
Not sure this is the final name yet, but I cannot abide such a non-descriptive placeholder!